### ISSUE TITLE
Prettier mangles indented HEREDOC with string interpolation

### ISSUE THREAD

**Issue by hackel** [prettier/plugin-php#1543]
**@prettier/plugin-php v0.14.3**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCADsAOj4gWMCG3iyVAYXQBbH2wAFAAkA-BdBQxAOVwBjKABlGABPNWswbAgAWgAmJxBoAEdBeGgA12V-PTkFEGIRCBgytgq2KoMAXxYoCngYdBQ0eCxJUDZsGDaRTtrMMEEJvBAAHgB+YzAAHUwtgBIADx9ZAAIAXiOli4ANAFkAGS2jx-OAQXnqfvgALzYMTAA+YA7PoAaygOAA2gByNhvD7fX6QgC67SWAHpXjQ4T8hv8Hk8brcANxbXYHY5nC5LAl4x5LDHvCBfbFYAFA9CgiHQ2GM+E4pEo9Hcpm-P5bAnEzAgdpAA)

```sh
# Options:
--phpVersion 7.4
```

**Input:**
```php
<?php

$xml = <<<XML
    <Authorization>{$tokens['authorization']}</Authorization>
    XML;

$xml = <<<XML
    <Authorization>{$tokens['authorization']}</Authorization>
XML;

```

**Output:**
```php
<?php

$xml = <<<XML
<Authorization>{$tokens['authorization']}thorization>
XML;

$xml = <<<XML
    <Authorization>{$tokens['authorization']}</Authorization>
XML;

```

**Expected behavior:**

I would expect the input to be returned unchanged.  This appears to be related to #1274, however this is much more serious as it is actually mangling the string.  It appears that it is stripping out the number of characters after the variable that the XML closing tag is indented.  There also seems to be some weird behaviour triggered by the `<` in the closing XML tag.  This does not affect NOWDOCs.

It is also still pulling the closing XML tag to the first column, which it shouldn't do with phpVersion >= 7.3.

**Comment by czosel** [comment]
@hackel Thanks for the bug report, this is indeed a serious issue!

**Comment by czosel** [comment]
Fixed and released as part of v0.15.2 :tada:
